src/Sentry/SentrySdk.cs

```diff
@@ -124,6 +125,7 @@ internal static IDisposable UseHub(IHub hub)
         public static void Close()
         {
             var oldHub = Interlocked.Exchange(ref _hub, DisabledHub.Instance);
+            ProcessInfo.Instance = null;
```

Comment:
Better do to this after disposing the Hub since on Dispose events are still being processed and flushed out to Sentry.